application/deed/utils.py

```diff
@@ -0,0 +1,32 @@
+import os
+from jsonschema.validators import validator_for
+import json
+
+
+def validate_helper(json_to_validate):
+    error_message = ""
+    error_list = sorted(_title_validator.iter_errors(json_to_validate),
+                        key=str, reverse=True)
+
+    for count, error in enumerate(error_list, start=1):
+        error_message += "Problem %s:\n\n%s\n\n" % (count, str(error))
+
+    return len(error_list), error_message
+
+
+def _load_json_schema():
+    script_path = os.path.dirname(os.path.realpath(__file__))
```

Comment:
As mentioned, readability is the main benefit, its much easier for someone to think of where a file is relative to application root rather than where it is relative to which file that is calling the code.